### PR TITLE
feat(ios): set up xcode project

### PR DESCRIPTION
**Pull request by qtran1997**
set up the ios xcode project

### CODE REVIEW

File: ios/.gitignore [hunk 1]
```diff
@@ -0,0 +1,90 @@
+# Xcode
```
Comment (edwindo) [inline ios/.gitignore:1]: Add .DS_Store files to gitignore + remove the existing ones.

**Review by edwindo — COMMENTED** [review]
There's 1 more .DS_Store in the top level directory :(

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,2 @@
+.DS_Store
```
Comment (edwindo) [inline .gitignore:1]: beast

File: ios/.gitignore (hunk 1 above)
Comment (qtran1997) [reply ios/.gitignore:1]: Done!

**Review by edwindo — APPROVED** [review]
Looks beast to me

**Pull request merged**